docs/cid-cmd.md

```diff
@@ -15,6 +15,20 @@ cid-cmd [tool parameters] [command] [command parameters]
 ### update
 
 ### delete
+Usage: cid-cmd delete [OPTIONS]
+
+Delete Dashboard and all dependencies unused by other CID-managed dasboards
+(including QuickSight datasets, Athena views and tables)
+
+Command options:
+   --dashboard-id TEXT                   QuickSight dashboard id (cudos, cost_intelligence_dashboard, kpi_dashboard, ta-organizational-view, trends-dashboard etc)
+   --athena-database TEXT                Athena database
+   
+Options:
+  -y, --yes            confirm all
+  -v, --verbose
+  --dashboard-id TEXT  QuickSight dashboard id
+  --help               Show this message and exit.
```

Comment:
@rrdog-aws thanks for the contribution
plese can you use the code quote markdown ?